public/css/bootstrap.min.css

```diff
@@ -0,0 +1,11 @@
+@import url("https://fonts.googleapis.com/css?family=Ubuntu:400,700");/*!
+ * bootswatch v3.3.7
+ * Homepage: http://bootswatch.com
+ * Copyright 2012-2017 Thomas Park
+ * Licensed under MIT
+ * Based on Bootstrap
+*//*!
+ * Bootstrap v3.3.7 (http://getbootstrap.com)
```

Comment:
Can we use the latest version of bootstrap?

Reply:
@ElizabethOkerio That could cause a breaking change. In the first place, it an adapted version of Bootstrap - see https://bootswatch.com/. Notice the import statement at the very top. The fonts used across the odata.org site are being imported. I'd vote for deferring upgrading the version of the Bootstrap CSS to a dedicated PR.